catalog/spatial/stanag4559/spatial-stanag4559-source/src/test/java/org/codice/ddf/spatial/ogc/stanag4559/catalog/source/client/Client.java

```diff
@@ -0,0 +1,429 @@
+
+        GetParametersRequest parametersRequest = productMgr.get_parameters(product,
+                desired_parameters,
+                properties);
+        System.out.println("Completing GetParameters Request ...");
+
+        DAGHolder dagHolder = new DAGHolder();
+        parametersRequest.complete(dagHolder);
+
+        DAG dag = dagHolder.value;
+        System.out.println("Resulting Parameters From Server :");
+        printDAGAttributes(dag);
+        System.out.println();
+    }
+
+    private static void get_related_file_types(Product product) throws Exception {
+        System.out.println("Sending Get Related File Types Request...");
+        String[] related_file_types = productMgr.get_related_file_types(product);
+        System.out.println("Related File Types : ");
+        for (int i = 0; i < related_file_types.length; i++) {
+            System.out.println(related_file_types[i]);
+        }
+        System.out.println();
+    }
+
+    private static void get_related_files(ORB orb, Product product) throws Exception {
+        System.out.println("Sending Get Related Files Request...");
+
+        FileLocation fileLocation = new FileLocation("", "", "", "", "");
+        NameValue[] properties = {new NameValue("", orb.create_any())};
+        Product[] products = {product};
+
+        GetRelatedFilesRequest relatedFilesRequest = productMgr.get_related_files(products,
+                fileLocation,
+                "",
+                properties);
+        System.out.println("Completing GetRelatedFilesRequest...");
+
+        NameListHolder locations = new NameListHolder();
+
+        relatedFilesRequest.complete(locations);
+
+        System.out.println("Location List : ");
+        String[] locationList = locations.value;
+        for (int i = 0; i < locationList.length; i++) {
+            System.out.println(locationList[i]);
+        }
+        System.out.println();
+    }
+
+    private static OrderContents createOrder(ORB orb) throws Exception {
+        NameName nameName[] = {new NameName("", "")};
+
+        TailoringSpec tailoringSpec = new TailoringSpec(nameName);
+        PackagingSpec pSpec = new PackagingSpec("", "");
+        AbsTime needByDate = new AbsTime(new Date((short) 2, (short) 10, (short) 16),
+                new Time((short) 10, (short) 0, (short) 0));
+
+        MediaType mTypes[] = {new MediaType("", (short) 1)};
+        String benums[] = {""};
+        AlterationSpec aSpec = new AlterationSpec("",
+                orb.create_any(),
+                new Rectangle(new Coordinate2d(1.1, 1.1), new Coordinate2d(2.2, 2.2)),
+                GeoRegionType.NULL_REGION);
+        Product product = new _ProductStub();
+
+        Destination destination = new Destination();
+        destination.e_dest("");
+
+        ProductDetails productDetails[] = {new ProductDetails(mTypes, benums, aSpec, product, "")};
+        DeliveryDetails deliveryDetails[] = {new DeliveryDetails(destination, "", "")};
+
```

Comment:
We can't use checkstyle on this package because it is mostly implementations of Java code generated from IDL, and it does not adhere to our checkstyle rules.